common/setups/lm/util.py

```diff
@@ -0,0 +1,55 @@
+__all__ = ["LmData"]
+
+from typing import Optional, Union
+
+from sisyphus import Path
+
+
+class LmData:
+    """
+    stores all data necessary for RETURNN LmDataset
+    """
+
+    def __init__(
+        self,
+        data_file: Path,
+        vocabulary_file: Path,
+        word_based: bool,
+        seq_order: str,
+        epoch_split: Union[int, float],
+        unknown_symbol: str = "<UNK>",
+        map_orthography_file: Union[str, Path] = "",
+        start_end_symbol: str = "<sb>",
+        data_type: Optional[str] = None,
+    ):
+        self.data_file = data_file
+        self.vocabulary_file = vocabulary_file
+        self.word_based = word_based
+        self.seq_order = seq_order
+        self.epoch_split = epoch_split
+        self.unknown_symbol = unknown_symbol
+        self.map_orthography_file = map_orthography_file
+        self.start_end_symbol = start_end_symbol
+        self.data_type = data_type
+
+    def get_data_type(self):
+        return self.data_type
+
+    def get_dataset(self):
```

Comment:
Hmm, we still have the problem of where dataset helpers are located. Maybe it would be good to collect them all under `i6_experiments/common/setups/returnn` like I wanted to have in the beginning (instead of returnn-common). But moving this is not hash critical, so we could do that later.

Reply:
I would postpone this for now.

Reply:
> I would postpone this for now.

Agreed. I think we could put this on the agenda for our Tuesday meeting